node/src/test_utils/recorder.rs

```diff
@@ -824,104 +826,133 @@ mod tests {
             NodeToUiMessage
         ));
         let respondent_addr = respondent.start();
-        let msg_1 = StartMessage {};
-        let counter_msg_1 = ScanForReceivables {
-            response_skeleton_opt: None,
-        };
-        // Also testing this convenient macro if it works fine
-        let setup_1 = setup_for_counter_msg_triggered_via_type_id!(
-            StartMessage,
-            counter_msg_1,
-            &respondent_addr
-        );
-        let counter_msg_2_strayed = StartMessage {};
-        let random_id = TypeId::of::<BlockchainBridge>();
-        let id_method_2 = MsgIdentification::ByType(random_id);
-        let setup_2 = SingleCounterMsgSetup::new(
-            random_id,
-            id_method_2,
-            vec![Box::new(SendableCounterMsgWithRecipient::new(
-                counter_msg_2_strayed,
-                respondent_addr.clone().recipient(),
-            ))],
-        );
-        let msg_3_unmatching = NewPublicIp {
-            new_ip: IpAddr::V4(Ipv4Addr::new(7, 7, 7, 7)),
-        };
-        let msg_3_type_id = msg_3_unmatching.type_id();
-        let counter_msg_3 = NodeToUiMessage {
-            target: MessageTarget::ClientId(4),
-            body: UiUnmarshalError {
-                message: "abc".to_string(),
-                bad_data: "456".to_string(),
-            }
-            .tmb(0),
-        };
-        let id_method_3 = MsgIdentification::ByMatch {
-            exemplar: Box::new(NewPublicIp { new_ip: make_ip(1) }),
+        // Case 1
+        let (trigger_message_1, cm_setup_1) = {
+            let trigger_msg = StartMessage {};
+            let counter_msg = ScanForReceivables {
+                response_skeleton_opt: None,
+            };
+            // Also testing this convenient macro if it works fine
+            (
+                trigger_msg,
+                setup_for_counter_msg_triggered_via_type_id!(
+                    StartMessage,
+                    counter_msg,
+                    &respondent_addr
+                ),
+            )
         };
-        let setup_3 = SingleCounterMsgSetup::new(
-            msg_3_type_id,
-            id_method_3,
-            vec![Box::new(SendableCounterMsgWithRecipient::new(
-                counter_msg_3,
-                respondent_addr.clone().recipient(),
-            ))],
-        );
-        let msg_4_matching = NewPublicIp {
-            new_ip: IpAddr::V4(Ipv4Addr::new(4, 5, 6, 7)),
+        // Case two
+        let cm_setup_2 = {
+            let counter_msg_strayed = StartMessage {};
+            let random_id = TypeId::of::<BlockchainBridge>();
+            let id_method = MsgIdentification::ByType(random_id);
```

Comment:
The `random_id` name is not good enough, it'll be better if we can point out that it's `screwed_id` for the trigger message. You'd like to give it a better name later on.